dnf/subject.py

```diff
@@ -77,7 +78,7 @@ def _query_flags(self):
 
     @property
     def filename_pattern(self):
-        return self.subj.pattern.find('/') != -1
+        return re.search(r"^\*?/", self.subj.pattern)
```

Comment:
This won't match patterns like ```*i*n/foo``` but let's assume that no one is going to try it... :)